src/spacel/model/app.py

```diff
@@ -24,6 +24,8 @@ def __init__(self, orbit, params=None):
         self.scheme = params.get('scheme', 'internet-facing')
         self.health_check = params.get('health_check', 'TCP:80')
         self.local_health_check = params.get('health_check', 'TCP:80')
+        self.loadbalancer = params.get('loadbalancer', 'true')
+        self.availability = params.get('availability', 'private')
```

Comment:
I like that!
`instance_availability`: `internal|multi-region|internet-facing`
`elb_availability`: `internal|multi-region|internet-facing|none`

`{instance,elb}_scheme` works too (potato/potatoe): the similar names and potential values are what feels warm/fuzzy.

`multi-region` is my `private multi-region` case from the above: we treat it like `internet-facing` in terms of Subnet allocation, but throw an error at https://github.com/pebble/spacel-provision/blob/master/src/spacel/provision/template/ingress_resource.py#L59 if you try to open it to the unwashed internets.